hack/update-api-docs.sh

```diff
@@ -7,30 +7,45 @@ set -o pipefail
 # Setup at https://github.com/ahmetb/gen-crd-api-reference-docs
 
 readonly SCRIPT_ROOT="$(git rev-parse --show-toplevel)"
+export GO111MODULE="on"
+go mod vendor
```

Comment:
`go mod vendor` has been done in Makefile, put it here to make sure `./hack/update-api-docs.sh` also works.